Add is/setVehicleWindowOpen

lopezloo opened the pull request:
So, this basically adds clientside function to control vehicle windows. Also I seen when ped start driveby as driver the window don't open, this fixes that. I redefined windows ID to be 0 - 6 but I'm not sure I do it proper place (game_sa).

Commit:
Add get/setVehicleWindowOpen

sbx320 commented on MTA10/mods/shared_logic/CClientPed.cpp:
```diff
@@ -5576,12 +5576,37 @@ void CClientPed::SetDoingGangDriveby ( bool bDriveby )
         {
-            bool bRight = ( GetOccupiedVehicleSeat ( ) % 2 == 0 ) ? false : true;
+            char cSeat = GetOccupiedVehicleSeat ( );
+            bool bRight = ( cSeat % 2 == 0 ) ? false : true;
```
cSeat % 2 == 0 is already a boolean expression.

cSeat % 2 != 0 has the same effect without the ternary operator

Commit:
Vehicle windows fixes

lopezloo commented:
Hm... I can do it but in that case function will return always true, even vehicle don't have specified window.

//edit
Maybe make this always return true? I seen setVehicleDoorState do that.

ccw808 commented:
I think that is acceptable.

Commit:
Made setVehicleWindowOpen works for unstreamed vehicles

Return always true (between 0 - 6 IDs) if vehicle is unstreamed, but
return true value if is streamed.

lopezloo commented:
Ok, done.

qaisjp commented:
Nice work! :+1:
